pipelines.adoc

```diff
@@ -47,35 +73,25 @@ Explore how to use pipelines to build and manage application stacks.
 
 . link:https://github.com/kabanero-io/kabanero-foundation[Kabanero foundation] must be installed on a supported Kubernetes deployment.
 
-. link:https://github.com/tektoncd/dashboard[A pipelines dashboard] is installed by default with Kabanero's Kubernetes operator. To find the pipelines dashboard URL, login to your cluster and run the `+oc get routes+` command or check in the console. You can also find the dashboard URL from the getting started page in the Kubernetes console.
+. link:https://github.com/tektoncd/dashboard[A pipelines dashboard] is installed by default with Kabanero's Kubernetes operator. To find the pipelines dashboard URL, login to your cluster and run the `+oc get routes+` command or in the Kabanero landing page. 
 
-. Dynamic volume provisioning or a persistent volume must be configured. See the following section for details.
+. A persistent volume must be configured. See the following section for details.
+
+. Secrets for the git repo (if private) and image repository
 
 === Getting started
 
 ==== Set up a persistent volume to run pipelines
 
-Pipelines require a configured volume that is used by the framework to share data across tasks. The `build task`, which uses `Buildah`, also requires a volume mount. The pipelinerun creates a Persistent Volume Claim (PVC) with a requirement for five GB of persistent volume.
-
-. Log in to your cluster. For example,
-+
-----
-oc login <master node IP>:8443
-----
-
-. Clone the link:https://github.com/kabanero-io/kabanero-pipelines[kabanero-pipeplines repo]
-+
-----
-git clone https://github.com/kabanero-io/kabanero-pipelines
-----
+Pipelines require a configured volume that is used by the framework to share data across tasks.  The pipelinerun creates a Persistent Volume Claim (PVC) with a requirement for five GB of persistent volume.
```

Comment:
Change
The pipelinerun creates
to
The pipelinerun command creates
(I assume it's a command)